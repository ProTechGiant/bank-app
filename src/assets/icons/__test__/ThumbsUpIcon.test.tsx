import { create } from "react-test-renderer";

import { ThumbsUpIcon } from "../ThumbsUpIcon";

describe("icon/ThumbsUpIcon", () => {
  it("renders a ThumbsUpIcon", () => {
    const result = create(<ThumbsUpIcon />);

    expect(result).toMatchSnapshot();
  });
});
