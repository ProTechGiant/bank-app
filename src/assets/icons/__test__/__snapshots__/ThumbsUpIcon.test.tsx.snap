// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon/ThumbsUpIcon renders a ThumbsUpIcon 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="18"
  bbWidth="18"
  fill="none"
  focusable={false}
  height={18}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 18,
        "width": 18,
      },
    ]
  }
  vbHeight={18}
  vbWidth={18}
  width={18}
  xmlns="http://www.w3.org/2000/svg"
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGPath
      d="M15.75 6.75H11.0175L11.73 3.3225L11.7525 3.0825C11.7525 2.775 11.625 2.49 11.4225 2.2875L10.6275 1.5L5.6925 6.4425C5.415 6.7125 5.25 7.0875 5.25 7.5V15C5.25 15.825 5.925 16.5 6.75 16.5H13.5C14.1225 16.5 14.655 16.125 14.88 15.585L17.145 10.2975C17.2125 10.125 17.25 9.945 17.25 9.75V8.25C17.25 7.425 16.575 6.75 15.75 6.75ZM15.75 9.75L13.5 15H6.75V7.5L10.005 4.245L9.1725 8.25H15.75V9.75ZM0.75 7.5H3.75V16.5H0.75V7.5Z"
      fill={
        {
          "payload": 4281216558,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
